### DISCUSSION TITLE
Deprecated options.

### DISCUSSION THREAD

**Discussion by KansaiUser** [pycaret/pycaret#3822] [Q&A]
I am just starting to learn Pycaret as part of a course on MLOps. However I am having a hard time running the tutorials because it seems that Pycaret versions are not compatible with older versions and there seems to be deprecated features. 

For example the "silent" option in setup is no longer supported. 

But my main question is what is the current equivalent of

```
exp_reg182 = setup(data=data, target='Price',session_id=123,
                                 normalize=True, transformation=True, transform_target=True,
                                 combine_rare_levels=True, rare_level_threshold=0.05,
                                 remove_multicollinearity=True, multicollinearity_threshold=0.95,
                                bin_numeric_features=['Cara Weight'] , log_experiment=True,
                               experiment_name='diamond',silent=True)
```
even if I delete silent, I got the error

setup() got an unexpected argument 'combine_rare_levels' and I suspect this is not the only one

**Comment by celestinoxp** [comment]
Hi,
pycaret is now in version 3 which brought a lot of updates, so maybe you can find it here in the documentation: [https://pycaret.readthedocs.io](https://pycaret.readthedocs.io)
example classification: [https://pycaret.readthedocs.io/en/latest/api/classification.html](https://pycaret.readthedocs.io/en/latest/api/classification.html)
